src/components/ReportWelcomeText.js

```diff
@@ -98,7 +97,12 @@ const ReportWelcomeText = (props) => {
                                 <Tooltip text={tooltip}>
                                     <Text
                                         style={[styles.textStrong]}
-                                        onPress={() => Navigation.navigate(ROUTES.getProfileRoute(participantAccountIDs[index]))}
+                                        onPress={() => {
+                                            const accountDetails = props.personalDetails[OptionsListUtils.addSMSDomainIfPhoneNumber(displayNamesWithTooltips[index].tooltip)];
```

Comment:
Wait, you couldn't replicate the error then?

Since that`index` doesn't come from the `participants` list or the `participantAccountIDs` one if you **refresh the page it shouldn't really work and point to the wrong profile**

Re-testing here

Reply:
Oh I see now, I was really confused haha

This is just the login, we are still using `props.personalDetails`, accepting the suggestion!